src/clients/inst_autoconfigure.rb

```diff
@@ -349,6 +350,11 @@ def main
       # Invoke SnapshotsFinish client to perform snapshots (if needed)
       WFM.CallFunction("snapshots_finish", ["Write"])
 
+      if ProductFeatures.GetBooleanFeature("globals", "save_y2logs")
+        Yast::SCR.Execute(Yast::Path.new(".target.bash"),
+                          "/usr/sbin/save_y2logs '#{Yast::Directory.logdir}/yast-installation-logs.tar.xz'")
```

Comment:
Can it happen that `save_y2logs` is not installed?

Reply:
The package dependencies require it.